implementations/rust/ockam/ockam_command/src/tcp/connection/create.rs

```diff
@@ -45,7 +42,9 @@ impl CreateCommand {
             OutputFormat::Plain => {
                 let from = &self.node_opts.from;
 
-                let to = response.socket_addr.parse::<SocketAddrV4>()?;
+                let to = response.socket_addr().map_err(|_| {
```

Comment:
Related to the other comment, if `socket_addr` returns an ockam_core::Error, then you could leave this unmodified.